test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -103,6 +104,14 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 
 		nodes := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
 		nodeCount = len(nodes.Items)
+		coreCount = 0
+		for _, node := range nodes.Items {
+			for resourceName, value := range node.Status.Capacity {
```

Comment:
nit: we don't need to iterate over all the resources, but can access CPU directly by key. Sorry for not catching it earlier